src/vconsole/vconsole-setup.c

```diff
@@ -213,46 +223,73 @@ static void font_copy_to_all_vcs(int fd, unsigned int vc_cnt) {
                 return;
         }
 
+        fontbuf = new(char, cfo.width * cfo.height * cfo.charcount / 8);
```

Comment:
given that this isn't really a char you allocate here, I'd just use plain "malloc()" as that returns "void*".